html/webappapis/microtask-queuing/queue-microtask-shadowrealm-callback-report-exception.window.js

```diff
@@ -0,0 +1,33 @@
+// META: title=Errors thrown by wrapped microtask in ShadowRealm
+
+setup({ allow_uncaught_exception: true });
+
+const realm = new ShadowRealm();
+
+const onerrorCalls = [];
+window.onerror = e => {
+  onerrorCalls.push("Window");
+};
+realm.evaluate(`(push) => {
+  onerror = function (e) {
+    const assertResult = e instanceof TypeError;
+    push(assertResult);
+  };
+}`)(assertResult => {
+  onerrorCalls.push("ShadowRealm");
+  assert_true(assertResult,
+    "exception should be converted to a fresh ShadowRealm TypeError")
+});
+
+async_test(t => {
+  window.onload = t.step_func(() => {
+    const task = () => { throw new Error("PASS"); };
```

Comment:
I'm not sure about the "PASS" here - that's not actually guaranteed to be used anywhere, is it? I would probably use any other string